apps/notepad/src/app/component/notes-list/note.model.ts

```diff
@@ -0,0 +1,13 @@
+import { Task } from '../note-details/task.model';
+
+export interface Note {
+  _id: string;
+  title: string;
+  content: string;
+  color?: string;
+  todoList?: Array<Task>;
+  createdAt: Date;
+  updatedAt?: Date;
+  attachment?: Blob;
+  label?: string;
```

Comment:
Improvement: Note might support multiple labels. Change the data type accordingly.